src/vs/workbench/browser/parts/panel/media/panelPart.css

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.monaco-workbench > .panel > .content > .panel {
+	height: 100%;
+}
+
+.monaco-workbench > .part.panel > .title {
+	display: block;
+	border-top: 1px solid #007ACC
```

Comment:
@bpasero status bar does the same thing, defines a default, and then clients like debug set it to orange. I do the same here.